grantnav/frontend/templates/search.html

```diff
@@ -115,15 +115,15 @@ <h3 class="indent"> Filter By <a id="filter_by_help_link" href="/help#filters"><
                 <div class="col-lg-5 col-md-6 no-padding-hor">
                   <div class="form-group form-group-sm">
                     <label for="min-amount">Smallest
-                    <input id="min-amount" name="min_amount" class="form-control" value="{{query.query.bool.filter.3.bool.should.range.amountAwarded.gte}}"/>
+                    <input id="min-amount" name="new_min_amount" class="form-control" value="{{query.query.bool.filter.3.bool.should.range.amountAwarded.gte}}"/>
```

Comment:
Is there a reason for the `new_` prefix?

Reply:
Did not want this form to have the same names as the params that end up in the URL.  Otherwise there is no way way for the backend to detect if it was this form being submitted or if the GET parameters were just comming from the URL.
After the form is submitted there is a redirect so the params in the URL turn to the correct ones.